tests/ops/test_stats.py

```diff
@@ -9,14 +9,18 @@
 
 def test_autocorrelation():
     x = torch.arange(10.)
-    assert_equal(autocorrelation(x),
+    with xfail_if_not_implemented():
```

Comment:
Thanks for this fix, also reported in #1548.

Reply:
+1